packages/demo/src/components/App.tsx

```diff
@@ -2,9 +2,19 @@ import { FunctionComponent } from 'react'
 import { CssBaseline, ThemeProvider } from '@mui/material'
 import { lightTheme } from '@storyblok/mui'
 import { FieldPluginDemo } from './FieldPluginDemo'
+import '@fontsource/roboto/300.css'
+import '@fontsource/roboto/400.css'
+import '@fontsource/roboto/500.css'
+import '@fontsource/roboto/700.css'
```

Comment:
Small thing: This had been forgotten when the demo app was created. Including here so that we don't forget it again.